LgiHdmiCec/LgiHdmiCec.cpp

```diff
@@ -967,7 +973,7 @@ namespace WPEFramework
         void LgiHdmiCec::onDeviceStatusChanged(IARM_EventId_t eventId, const void* data_ptr, size_t len)
         {
             assert(data_ptr != NULL);
```

Comment:
I would add: 
std::lock_guard<std::mutex> guard(m_mutex);

to protect accessed members: cecEnableStatus, m_scan_id etc...

As discussed this will be done in speparate ticket.